source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -135,10 +136,28 @@ template <typename LogRequest, typename LogResponse> class GrpcAccessLogClient {
     return true;
   }
 
+  Http::AsyncClient::StreamOptions createStreamOptionsForRetry() {
+    auto opt = Http::AsyncClient::StreamOptions();
+
+    if (!grpc_stream_retry_policy_) {
+      return opt;
+    }
+
+    envoy::config::route::v3::RetryPolicy retry_policy;
+    retry_policy.mutable_num_retries()->set_value(
```

Comment:
I think that only `num_retries` from the retry policy is being used needs to be documented.

Reply:
added

Reply:
Why only `num_retries` applies?

Reply:
I think that we don't need backoff strategy in our usecase. Our usecase is just to reconnect other cluster periodically, not to reconnect same cluster. So to speak, we don't expect that to overwhelm periodical pressure in current usecase, then backoff is not needed right now. This is why I add only `num_retries`.

Reply:
My expectation to use this feature is to configure `num_retries` as the same number of cluster hosts.

Reply:
added backoff